images-list

```diff
@@ -214,6 +214,8 @@ library/nginx rancher/mirrored-library-nginx 1.21.1-alpine
 library/traefik rancher/library-traefik 2.4.2
 library/traefik rancher/library-traefik 2.4.8
 library/traefik rancher/library-traefik 2.4.9
+library/traefik rancher/mirrored-library-traefik 2.4.8
+library/traefik rancher/mirrored-library-traefik 2.4.9
```

Comment:
Why both ? I don't see 2.4.9 in the other PR.

Reply:
I tried to match the version between the old `rancher/library-traefik` and new `rancher/mirrored-library-traefik`, so I add all versions from 2.4.8 in case we will need 2.4.9 in the near future. 

But I can remove it for now and add it when it is needed.